Core/Core/Logging/Analytics.cs

```diff
@@ -67,7 +67,17 @@ public enum Events
     /// <summary>
     /// Event triggered by the Mapping Tool
     /// </summary>
-    MappingsAction
+    MappingsAction,
+
+    /// <summary>
+    /// Event triggered on send, capturing the object type counts of the sent commit
+    /// </summary>
+    SendObjectReport,
```

Comment:
Can we just make these without setup or do we need to setup on MixPanel side, presumably it's either automatic or we did the setup? Just asking? Also, do we know what part of our estate is release and what part is WIP?